library/pathlib.po

```diff
@@ -41,165 +47,215 @@ msgid ""
 msgid ""
 "If you want to manipulate Windows paths on a Unix machine (or vice versa). "
 "You cannot instantiate a :class:`WindowsPath` when running on Unix, but you "
 "can instantiate :class:`PureWindowsPath`."
 msgstr ""
+"Si vous voulez manipuler des chemins Windows sur une machine Unix (ou vice "
+"versa). Vous ne pouvez pas instancier un :class:`WindowsPath` quand vous "
+"êtes sous Unix, mais vous pouvez instancier :class:`PureWindowsPath`."
 
 #: ../Doc/library/pathlib.rst:34
 msgid ""
 "You want to make sure that your code only manipulates paths without actually "
 "accessing the OS. In this case, instantiating one of the pure classes may be "
 "useful since those simply don't have any OS-accessing operations."
 msgstr ""
+"Vous voulez être sur que votre code manipule uniquement sans réellement "
+"accéder au système d'exploitation. Dans ce cas, instancier une de ces "
+"classes pures peut être utile puisqu'elle ne peut simplement accéder aux "
+"opérations du système d'exploitation."
 
 #: ../Doc/library/pathlib.rst:39
 msgid ":pep:`428`: The pathlib module -- object-oriented filesystem paths."
-msgstr ""
+msgstr ":pep:`428`: Le module pathlib -- système de fichiers orienté objet."
 
 #: ../Doc/library/pathlib.rst:42
 msgid ""
 "For low-level path manipulation on strings, you can also use the :mod:`os."
 "path` module."
 msgstr ""
+"Pour de la manipulation de chemins bas niveau avec des chaînes de "
+"caractères, vous pouvez aussi utiliser le module :mod:`os.path`."
 
 #: ../Doc/library/pathlib.rst:47
 msgid "Basic use"
-msgstr ""
+msgstr "Utilisation basique"
 
 #: ../Doc/library/pathlib.rst:49
 msgid "Importing the main class::"
-msgstr ""
+msgstr "Importer la classe principale ::"
 
 #: ../Doc/library/pathlib.rst:53
 msgid "Listing subdirectories::"
-msgstr ""
+msgstr "Lister les sous-dossiers ::"
 
 #: ../Doc/library/pathlib.rst:60
 msgid "Listing Python source files in this directory tree::"
 msgstr ""
+"Lister les fichiers source Python dans cette arborescence de dossiers ::"
 
 #: ../Doc/library/pathlib.rst:67
 msgid "Navigating inside a directory tree::"
-msgstr ""
+msgstr "Naviguer à l'intérieur d'une arborescence de dossiers ::"
 
 #: ../Doc/library/pathlib.rst:76
 msgid "Querying path properties::"
-msgstr ""
+msgstr "Récupérer les propriétés de chemin ::"
 
 #: ../Doc/library/pathlib.rst:83
 msgid "Opening a file::"
-msgstr ""
+msgstr "Ouvrir un fichier ::"
 
 #: ../Doc/library/pathlib.rst:93
 msgid "Pure paths"
-msgstr ""
+msgstr "Chemins purs"
 
 #: ../Doc/library/pathlib.rst:95
 msgid ""
 "Pure path objects provide path-handling operations which don't actually "
 "access a filesystem.  There are three ways to access these classes, which we "
 "also call *flavours*:"
 msgstr ""
+"Les objets chemins purs fournissent les opérations de gestion de chemin qui "
+"n'accèdent pas réellement au système de fichiers. Il y a trois façons "
+"d'accéder à ces classes que nous appelons aussi *saveurs* :"
```

Comment:
I don't think we can translate with *saveurs* here, maybe *familles*?

Reply:
Yeah. I've already the usage of *saveurs* in some other docs, but *familles* is easier to understand.